package.json

```diff
@@ -48,12 +50,14 @@
   },
   "dependencies": {
     "@canonical/maraca": "^0.1.2",
+    "hoist-non-react-statics": "^3.0.1",
```

Comment:
This is a little weird being here, I wonder if we shouldn't vendor it along with the modified click-outside code.

Reply:
Yes, not in the scope of this branch.